lib/Factory/DisplayFactory.php

```diff
@@ -224,8 +224,8 @@ public function query($sortOrder = null, $filterBy = [])
                   display.broadCastAddress,
                   display.secureOn,
                   display.cidr,
-                  ' . $functionPrefix . 'X(display.GeoLocation) AS latitude,
-                  ' . $functionPrefix . 'Y(display.GeoLocation) AS longitude,
+                  IFNULL( ' . $functionPrefix . 'X(display.GeoLocation), ' . $this->config->getSetting('DEFAULT_LAT'). ') AS latitude,
```

Comment:
I don't think we can do this here, because that would mean that the displays in the grid also show as the default lat/long. It is very useful to see which displays actually have a lat/long, so we don't want to affect the grid.

Instead you could output the default lat/long to the page, and then handle this client side when you pass the pin to the map?